understanding/22/dragging-movements.html

```diff
@@ -41,7 +41,11 @@ <h2>Intent of Dragging Movements</h2>
       
     <p>This Success Criterion applies to dragging motions as opposed to pointer gestures which are covered in Success Criterion 2.5.1 Pointer Gestures. Pointer gestures include directional path-based as well as multi-point gestures. In contrast, for dragging motions, only the start and end point of the motion matters, not the actual path.</p>
 
-		<p>2.1.1 Keyboard and 2.1.3 Keyboard (No Exception) require dragging features to be keyboard accessible, however, it is possible to create an interface that works with dragging and keyboard controls that does not work using only clicks or taps.</p>
+      <p>2.1.1 Keyboard and 2.1.3 Keyboard (No Exception) require dragging features to be keyboard accessible, however, it is possible to create an interface that works with dragging and keyboard controls that does not work using only clicks or taps.</p>
+
+      <p>There may be cases where a dragging movement is essential, for example, drawing a freeform figure, or playing a game where the aim is to follow a particular path as accurately as possible. The criterion has an exception for these cases where the task requires dragging.</p>
```

Comment:
isn't "follow a particular path" by definition a "path-based gesture", so not a "dragging movement"?

Reply:
Hmm, it's a term clash. I'll change that to 'track' to avoid confusion.

Reply:
+1 to @patrickhlauke, a few of us are also still having trouble understanding how this differs from 2.5.1: Pointer Gestures from 2.1 and just want to make sure we are properly comprehending. If we've understood correctly, the difference is that this dragging SC involves pointer interaction where _only_ the endpoints matter vs. the 2.5.1 PG SC is interaction where _not_ just the endpoints matter.

Would it be possible to further expand on the "follow the path" portion in the game example? Is that meant to illustrate percentage of something being correctly traced/colored (e.g. 96% accuracy), which seems like a path-based gesture? Or rather an in-game character traversing (which could have a single pointer arrow key or maybe even a long press to "move them"), which doesn't seem to be path-dependent?

Thank you for considering!